int8 edgeEnhancementSet
---
int8 edgeEnhancementGet
string edgeEnhancementGetString